resources/zib/zib-O2Saturation.xml

```diff
@@ -0,0 +1,163 @@
+      <alias value="O2Saturatie" />
+      <mapping>
+        <identity value="zib-o2saturation-v3.1-2020EN" />
+        <map value="NL-CM:12.10.1" />
+        <comment value="O2Saturation" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="2708-6" />
+        </coding>
+        <coding>

```

Comment:
Are we sure we want mandate the use both loinc codes? Would the first not be enough?

Reply:
2708-6 is required by Vital Signs (Oxygen saturation in Arterial blood), 59408-5 is the specificity added by the zib (... by Pulse oximetry). Leaving the first one out would break Vital Signs compatibility, leaving the second one out would make this profile wider than the zib.

Reply:
ok! Thnx.